python/ray/experimental/gcs_flush_policy.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import logging
+import time
+
+import ray
+import ray.cloudpickle as pickle
+
+# Set up logging.
+logging.basicConfig()
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
```

Comment:
This doesn't seem like the right place to set up a global logging variable.

Reply:
I removed these.